### PR TITLE
Feature - store and fetch filter data on history state

### PR DESCRIPTION
**Pull request by yonikid15**
# Notes
- Should store and pass `poolCandidateFilter` data from the `<SearchForm />` to the `<RequestPage />`.
- Moves the shared `<Toast />` component to the common folder.
- Removes the helper folder from admin, since it's already found in common. 
- Ensures users selected fields are not reset when clicking back to search form

### CODE REVIEW

File: common/src/helpers/router.tsx [hunk 1]
```diff
@@ -59,6 +59,14 @@ export const redirect = (url: string | Partial<Path>): void => {
   HISTORY.replace(url);
 };
 
+export const navigateBack = (): void => {
```
Comment (petertgiles) [inline common/src/helpers/router.tsx:62]: Uh oh, you got a conflict from #1196 too.  Could you resolve this?

Comment (yonikid15) [reply common/src/helpers/router.tsx:71]: Should be good to go now.

**Review by petertgiles — CHANGES_REQUESTED** [review]
This is looking great, Yoni.  I noticed, though, that the education level doesn't seem to be transferring properly.  I selected work/education and got required diploma on the result page.  Could you check it out?
image

**Comment by yonikid15** [comment]
> This is looking great, Yoni. I noticed, though, that the education level doesn't seem to be transferring properly. I selected work/education and got required diploma on the result page. Could you check it out? image

Nice catch! I guess the form values returns the values as strings so converting it to a boolean always resulted in it being true. I made a fix for it now.

**Review by petertgiles — CHANGES_REQUESTED** [review]
Wow, this was a big one!  It all looks good to me except for a stray console.log left in.  Great work!

**Review by petertgiles — APPROVED** [review]
I just removed the console.log.

**Pull request merged**